tests/sentry/api/endpoints/test_organization_member_team_details.py

```diff
@@ -436,6 +523,31 @@ def test_admin_on_team_can_remove_members(self):
             team=self.team, organizationmember=self.manager_on_team
         ).exists()
 
+        # owner
```

Comment:
Could you add some detail to this comment?

Reply:
This is following the convention for the tests, it's specifying the org admin on team can remove the owner who's also on the same team